src/primitives/bool.ts

```diff
@@ -0,0 +1,28 @@
+import { type Options, SizedType } from "../types/mod.ts";
+
+export class Bool extends SizedType<boolean> {
+  constructor() {
+    super(1, 1);
+  }
+
+  readUnaligned(dt: DataView, options: Options = { byteOffset: 0 }): boolean {
+    super.rangeCheck(dt.byteLength, options.byteOffset);
+
+    const value = !!dt.getInt8(options.byteOffset);
```

Comment:
```suggestion
    const value = Boolean(dt.getInt8(options.byteOffset));
```
Less JavaScripty === more readability